app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -606,6 +611,20 @@ def configure!(scoring_configs:)
         end
       end
 
+      def inject_config_ab_test(valid_method_name, scoring_config)
+        # Change a value if the strategy is not original
+        return scoring_config unless valid_method_name == :comments_count_factor && @strategy != "original"
```

Comment:
Could you break these into separate guard clauses?  I find I stumble on `unless ... && !=` structures.

```
return scoring_config unless valid_method_name == :comments_count_factor
return scoring_config if @strategy == "original"
```

Reply:
Done